libkpod/containerserver.go

```diff
@@ -113,7 +120,147 @@ func New(config *Config) (*ContainerServer, error) {
 			sandboxes:  make(map[string]*sandbox.Sandbox),
 		},
 		config: config,
-	}, nil
+	}
+
+	return c, c.Update()
+}
+
+// Update makes changes to the server's state (lists of pods and containers) to
+// reflect the list of pods and containers that are stored on disk, possibly
+// having been modified by other parties
+func (c *ContainerServer) Update() error {
```

Comment:
They were, minus the inclusion of pods.  Although I'm remembering now that you added support for pods in the ContainerServer, so that didn't need to be removed.  I'll go back and simply move Update() and LoadContainer() into ContainerServer{}